### ISSUE TITLE
expval returns ExpectationMP

### ISSUE THREAD

**Issue by kayuksel** [YeonwooSung/EvoTorch-Tutorial#1]
Hello,

expal function returns ExpectationMP instead of a torch float or double.

I have also tried the code from PennyLane but that was not the case.

Do you know what is causing this or how to fix this?

**Comment by YeonwooSung** [comment]
Hi,

When you say "expal function returns ExpectationMP instead of a torch float or double", you mean the expval function not the expal right?
As far as I know, the expval function is a function that is provided by "PennyLane", neither PyTorch nor evotorch.
And if you read the [document of expval function](https://docs.pennylane.ai/en/stable/code/api/pennylane.expval.html), you could find that the return type of the expval function is ExpectationMP, not torch dtype.

**Comment by kayuksel** [comment]
I get torch tensors as an output when I change the backend to torch, actually. However, for the input of expval, they do an internal numpy conversion, which is not possible to make for the tensors having requires_grad. I wanted to try a gradient-based optimizer.

**Comment by YeonwooSung** [comment]
If that's the case, then you could apply the torch interface that is provided by PennyLane.
Please refer to [this reference link](https://docs.pennylane.ai/en/stable/introduction/interfaces/torch.html).
